enterprise/server/workflow/service/service_test.go

```diff
@@ -299,8 +300,10 @@ func TestWebhook_UntrustedPullRequest_StartsUntrustedWorkflow(t *testing.T) {
 	execClient := &fakeExecutionClient{}
 	te.SetRemoteExecutionClient(execClient)
 	ws := te.GetWorkflowService()
-	url := testhttp.StartServer(t, ws)
-	flags.Set(t, "app.build_buddy_url", url)
+	appUrl := testhttp.StartServer(t, ws)
```

Comment:
can StartServer return URL instead of string? then we can remove some boilerplate code here.

Reply:
Done!